src/main/scala/gemmini/ReservationStation.scala

```diff
@@ -404,33 +374,50 @@ class ReservationStation[T <: Data : Arithmetic, U <: Data, V <: Data](config: G
   }
 
   // Issue commands which are ready to be issued
-  Seq((ldq, io.issue.ld), (stq, io.issue.st), (exq, io.issue.ex)).foreach { case (q, io) =>
-    val issue_valids = entries.map(e => e.valid && e.bits.ready() && !e.bits.issued && e.bits.q === q)
+  Seq((ldq, io.issue.ld, entries_ld), (stq, io.issue.st, entries_st), (exq, io.issue.ex, entries_ex))
```

Comment:
note to self: should've used the previous version of this, since the new ID scheme wasn't applied to elsewhere. if something failed, it's here